docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md

```diff
@@ -0,0 +1,80 @@
+# Play Slick 0.9 Migration Guide
+
+This is a guide for migrating from Play Slick 0.8 to Play Slick 0.9.
+
+It assumes you have already migrated your project to use Play 2.4 (see [Play 2.4 Migration Guide]), that you have read the [Slick 3 documentation], and are ready to migrate your Play application to use the new Slick Database I/O Actions API.
+
+## Build changes
+
+Update the Play Slick dependency in your sbt build to `0.9.0`:
+
+```sbt
+"com.typesafe.play" %% "play-slick" % "0.9.0"
+```
+
+[Play 2.4 Migration Guide]: https://www.playframework.com/documentation/2.4.x/Migration24)
+[Slick 3 documentation]: http://slick.typesafe.com/docs/
+
+## Database configuration
+
+In Slick 2.1 (and earlier) you used to configure Slick datasources exactly like you would configure Play JDBC datasources. This is no longer the case with Slick 3, and the following configuration will in fact be simply ignored by Play Slick:
```

Comment:
This paragraph needs to be reformulated, as the changes explained here are not about Slick but about Play Slick.